CHANGES.rst

```diff
@@ -13,6 +13,15 @@ Changelog
 
 .. towncrier release notes start
 
+1.0.0.post1 (2020-07-16)
```

Comment:
I think we shouldn't include a changelog with a post release because the idea of the post release is that it's just packaging.

After thinking this over more, I think we should file a bug that pulp-certguard is not compatible with pulpcore 3.5 and fix that bug by release pulp-certguard 1.0.1. The changelog would contain a single change in it then...

@fao89 wdyt?

Reply:
sounds good!

Reply:
https://pulp.plan.io/issues/7177

Reply:
https://github.com/pulp/pulp-certguard/pull/74